test/scripts/e2e_subs/e2e-app-other.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+date '+keyreg-teal-test start %Y%m%d_%H%M%S'
+
+set -e
+set -x
+set -o pipefail
+export SHELLOPTS
+
+WALLET=$1
+
+gcmd="goal -w ${WALLET}"
+
+ACCOUNT=$(${gcmd} account list|awk '{ print $3 }')
+
+# approval program
+printf '#pragma version 2\nint 1' > "${TEMPDIR}/simple.teal"
+PROGRAM=($(${gcmd} clerk compile "${TEMPDIR}/simple.teal"))
```

Comment:
isn't too many parenthesis?

Reply:
This statement is not needed anyway. Thanks for pointing it out.